packages/cmf/src/sagas/documentTitle.test.js

```diff
@@ -0,0 +1,66 @@
+import { takeLatest, take } from 'redux-saga/effects';
```

Comment:
There's a test file already in `cmf\__tests__\sagas\documentTitle.test.js`, maybe these tests could be moved there?